platform/darwin/src/MGLMapSnapshotter.h

```diff
@@ -139,6 +158,38 @@ MGL_EXPORT
 - (void)cancel;
 
 /**
+ The zoom level.
+ 
+ The default zoom level is 0. If this property is non-zero and the camera property
+ is non-nil, the camera’s altitude is ignored in favor of this property’s value. 
+ */
+@property (nonatomic) double zoomLevel;
+
+/**
+ A camera representing the viewport visible in the snapshot.
+ 
+ This overwrites the coordinate bounds if set.
```

Comment:
Similarly to https://github.com/mapbox/mapbox-gl-native/pull/10163#discussion_r143601451, we should try to be consistent with the corresponding properties of MGLMapSnapshotOptions:

https://github.com/mapbox/mapbox-gl-native/blob/d913f4be79f2970c0ba95c297816632241eb374b/platform/darwin/src/MGLMapSnapshotter.h#L40-L44
https://github.com/mapbox/mapbox-gl-native/blob/d913f4be79f2970c0ba95c297816632241eb374b/platform/darwin/src/MGLMapSnapshotter.h#L49-L52

etc.